packages/freezed_annotation/lib/freezed_annotation.dart

```diff
@@ -200,3 +200,9 @@ class With {
   final Type? type;
   final String? stringType;
 }
+
+class FreezedUnionValue {
```

Comment:
We'll need this commented and documented in the readme

Reply:
Sure, I just wanted to make sure if you're ok with this approach first.